app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -93,4 +110,19 @@ export class CreatorSettingsController extends Controller {
       brightness(color, 0.85),
     );
   }
+
+  /**
+   * Prevents a submission of the form if the
+   * color contrast is low.
+   *
+   * @param {Event} event
+   */
+  formValidations(event) {
+    const { value: color } = this.brandColorTarget;
+    if (isLowContrast(color)) {
+      this.colorContrastErrorTarget.classList.remove('hidden');
+      //  we don't want the form to submit if the contrast is low
+      event.preventDefault();
```

Comment:
Nitpick (non-blocking): Put `event.preventDefault();` at the beginning of the code block.

Reply:
✅